layouts/shortcodes/media-consent.html

```diff
@@ -1,2 +1,2 @@
 <h2>Media Consent</h2>
-<p>In order to take photos and videos for promotional use (mainly on our website), we are required to collect a media consent release. We therefore require everyone (including team leaders, competitors and guests) who attends the the competition to complete a media consent form. Under 18s need a parent or guardian to sign on their behalf.</p>
+<p>In order to take photos and videos for promotional use (mainly on our website), we are required to collect a media consent release. We therefore require everyone (including team leaders, competitors and guests) who attends to complete a media consent form. Under 18s need a parent or guardian to sign on their behalf.</p>
```

Comment:
Our policy has changed this since was written, but since it's new in this PR let's fix it here: "Under 18s" -> "Under 16s".